.env.example

```diff
@@ -1,12 +1,19 @@
 ## thanos notification env
+export NETWORK=
+
 export L1_RPC=http://localhost:8545
 export L1_WS_RPC=ws://localhost:8546
 export L2_RPC=http://localhost:9545
 export L2_WS_RPC=ws://localhost:9546
-export L1_STANDARD_BIRDGE=
-export L2_STANDARD_BIRDGE=0x4200000000000000000000000000000000000010
-export L1_CROSS_DOMAIN_MESSENGER=
-export L2_CROSS_DOMAIN_MESSENGER=0x4200000000000000000000000000000000000007
-export L2_TO_L1_MESSAGE_PASSER=0x4200000000000000000000000000000000000016
-export OPTIMISM_PORTAL=
-export TRANSFER_ADDRESSES=0xC7844340d14deAedfDD2f2dD9360c336661b2F0A
\ No newline at end of file
+
+export L1_STANDARD_BRIDGE=
+export L2_STANDARD_BRIDGE=0x4200000000000000000000000000000000000010
+
+export SLACK_URL=
+
+export L1_EXPLORER_URL=
+export L2_EXPLORER_URL=
+
+export OFF=0
+
+export TOKEN_ADDRESSES=
```

Comment:
@nguyenzung @xxeonge 
Then, is it possible to supply notification for all kind of ERC20 token without TOKEN_ADDRESSES ?

Reply:
https://github.com/tokamak-network/tokamak-thanos-event-listener/blob/73d5307fdeb85fc5b7b8066636855cb0a782ac33/internal/app/thanos-notif/app.go#L119

Is it possible to add the call logic if there is no TokenInfo ? 🙇🏼‍♂️

Reply:
@rlgns98kr @xxeonge 
What do you think if we define TOKEN_ADDRESSES like this: 
Option 1: TOKEN_ADDRESSES= [{"TON": "<address of TON"}, {"USDC":"<address of USDC>"}, ...]
Option 1: TOKEN_ADDRESSES= "<address of TON>":"<address of USDC>":...

Reply:
In fact, I don't know about the need for the env. I think we can get the token address in the events. If we have the tokenInfo of the address, we can use it, or we can use contract call to token address and save it in tokenInfo!

Reply:
Ah, you are right, Steven
Actually, I thought it is used for filtering, because we only monitor some tokens.

Reply:
Oh, I'm sorry. I forgot it... Now, I remember that we were developed!

Okay, but if we didn't filter to use the token address like this code, I think we can supply noti for dep/with of all kinds of erc20 adding the updateTokenInfo logic!

Reply:
Yes, you are right. Your solution works for all ERC20 tokens

Reply:
I saw that Sia listed some Tokens on source code, so I thought it is for monitoring.

Reply:
Please check if my understanding is correct. @rlgns98kr @nguyenzung 

Rather than using env to obtain symbols and decimals of some tokens, obtain tokenInfo for all ERC20 by obtaining token addresses from events. It is correct?

Actually, My plan was to only put token addresses supported by Thanos into TOKEN_ADDRESSES and search for symbols and decimals ..! 🥹

If my understanding is correct, I will edit it as you advised.

Reply:
@xxeonge @nguyenzung 
Oh, I'm sorry for the confusion! 🙇🏼‍♂️🥹🙇🏼‍♂️🙇🏼‍♂️🙇🏼‍♂️

I think it is correct to use the ENV. I concern about the memory leaks....
Okay, I think the code is great!